templates/web/base/waste/dd_cancelled.html

```diff
@@ -0,0 +1,47 @@
+[% INCLUDE header.html %]
+
+<div class="govuk-error-summary">
+    <h1 class="govuk-panel__title">
+        Payment cancelled
+    </h1>
+    <div class="govuk-panel__body">
+        <p>
+        You cancelled your attempt to pay by Direct Debit.
+        </p>
+    </div>
+</div>
+
+[% IF report %]
+<form name="post" method="post" action="[% payment_details.dd_push_url %]/pages/registration.aspx?.Client=[% payment_details.dd_client_id %]&.PaymentPlans=[% payment_details.dd_payment_plan %]">
+    <input type="hidden" name="txtTitle" id="txtTitle"  value="[% title %]" />
```

Comment:
You can probably guess what I'm going to say... Isn't this (at least all the hidden fields) identical to the form in dd.html?